client/src/main/scala/org/http4s/client/PoolManager.scala

```diff
@@ -21,41 +23,36 @@ private final class PoolManager[F[_], A <: Connection[F]](
 
   private var isClosed = false
   private var curTotal = 0
-  private var allocated = mutable.Map.empty[RequestKey, Int]
-  private val idleQueue = mutable.Map.empty[RequestKey, mutable.Queue[A]]
+  private val allocated = mutable.Map.empty[RequestKey, Int]
+  private val idleQueues = mutable.Map.empty[RequestKey, mutable.Queue[A]]
   private val waitQueue = mutable.Queue.empty[Waiting]
 
   private def stats =
-    s"allocated=$allocated idleQueue.size=${idleQueue.size} waitQueue.size=${waitQueue.size}"
+    s"allocated=$allocated idleQueues.size=${idleQueues.size} waitQueue.size=${waitQueue.size} maxWaitQueueLimit=$maxWaitQueueLimit"
```

Comment:
Is this going to be an untenable toString, rendering an arbitrarily large map for `allocated`?

Reply:
Yeah, should we make a public method Key -> Stat for users?

Reply:
Yes, we need to put some thought into client metrics in general.  It has poor observability at the moment.